### ISSUE TITLE
Docker latest-alpine (15.9.2) stopped with errors

### ISSUE THREAD

**Issue by Frepke** [coderaiser/cloudcmd#337]
* **Version** (`cloudcmd -v`): docker latest-alpine (15.9.2)
* **Node Version** `node -v`: 
* **OS** (`uname -a` on Linux): Debian Buster
* **Browser name/version**: all
* **Used Command Line Parameters**: 
* **Changed Config**: 

When I upgrade to the latest docker alpine image, the container will not start.
There are a lot of errors in the logfile:
```
    today at 07:42  root: /mnt/fs
    today at 07:42  Loading linux musl addon Error: Cannot find module '@node-rs/bcrypt-linux-musl'
    today at 07:42  Require stack:
    today at 07:42  - /usr/src/app/node_modules/@node-rs/helper/lib/loader.js
    today at 07:42  - /usr/src/app/node_modules/@node-rs/helper/lib/index.js
    today at 07:42  - /usr/src/app/node_modules/@node-rs/bcrypt/index.js
    today at 07:42  - /usr/src/app/node_modules/http-auth/src/auth/basic.js
    today at 07:42  - /usr/src/app/node_modules/http-auth/src/http-auth.js
    today at 07:42  - /usr/src/app/server/auth.js
    today at 07:42  - /usr/src/app/server/cloudcmd.js
    today at 07:42  - /usr/src/app/server/server.js, Loading linux addon Error: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /usr/src/app/node_modules/@node-rs/bcrypt-linux/bcrypt.linux.node) Promise {
    today at 07:42    <rejected> TypeError: Loading linux musl addon Error: Cannot find module '@node-rs/bcrypt-linux-musl'
    today at 07:42    Require stack:
    today at 07:42    - /usr/src/app/node_modules/@node-rs/helper/lib/loader.js
    today at 07:42    - /usr/src/app/node_modules/@node-rs/helper/lib/index.js
    today at 07:42    - /usr/src/app/node_modules/@node-rs/bcrypt/index.js
    today at 07:42    - /usr/src/app/node_modules/http-auth/src/auth/basic.js
    today at 07:42    - /usr/src/app/node_modules/http-auth/src/http-auth.js
    today at 07:42    - /usr/src/app/server/auth.js
    today at 07:42    - /usr/src/app/server/cloudcmd.js
    today at 07:42    - /usr/src/app/server/server.js, Loading linux addon Error: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /usr/src/app/node_modules/@node-rs/bcrypt-linux/bcrypt.linux.node)
    today at 07:42        at loadBinding (/usr/src/app/node_modules/@node-rs/helper/lib/loader.js:45:27)
    today at 07:42        at Object.<anonymous> (/usr/src/app/node_modules/@node-rs/bcrypt/index.js:3:17)
    today at 07:42        at Module._compile (node:internal/modules/cjs/loader:1092:14)
    today at 07:42        at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    today at 07:42        at Module.load (node:internal/modules/cjs/loader:972:32)
    today at 07:42        at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    today at 07:42        at Module.require (node:internal/modules/cjs/loader:996:19)
    today at 07:42        at require (node:internal/modules/cjs/helpers:92:18)
    today at 07:42        at Object.<anonymous> (/usr/src/app/node_modules/http-auth/src/auth/basic.js:16:16)
    today at 07:42        at Module._compile (node:internal/modules/cjs/loader:1092:14)
    today at 07:42  }
    today at 07:42 Container stopped
```
When I revert back to 15.9.1-alpine, everything is fine again.

**Comment by coderaiser** [comment]
Thank you :), that's relates to https://github.com/napi-rs/node-rs/issues/361

**Comment by coderaiser** [comment]
Fixed with 1801689e, landed in [v15.9.3](https://github.com/coderaiser/cloudcmd/releases/tag/v15.9.3) 🎉 .
Is it works for you?

**Comment by Frepke** [comment]
Yes, it's working again 👍
